### PR TITLE
Revised Test vectors text

### PR DESCRIPTION
**Pull request by Wind4Greg**
Removed "to be written" section. Cleaned up wording a bit.

***
<a href="https://pr-preview.s3.amazonaws.com/Wind4Greg/di-eddsa-2020/pull/24.html" title="Last updated on Feb 24, 2023, 6:19 PM UTC (6dc0118)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-di-eddsa/24/5a3b66a...Wind4Greg:6dc0118.html" title="Last updated on Feb 24, 2023, 6:19 PM UTC (6dc0118)">Diff</a>

### CODE REVIEW

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-02-08-vcwg#section3-6) on 2023-02-08

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.6. Revised Test vectors text (pr vc-di-eddsa#24)

_See github pull request [vc-di-eddsa#24](https://github.com/w3c/vc-di-eddsa/pull/24)._




**Manu Sporny:** moving on to edwards curve suite.  
… p.r. to update test vectors.  
… good job, but not a working group member.  
… doesn't look like any ipr issues. needs to be cleared for IP.  
… thats it.  

</details>

**Review by msporny — APPROVED** [review]

File: index.html
Comment (TallTed) [inline index.html]: Shifted to (American) English word order.
```suggestion
for signing and the public key made available for verification. The 
[[MULTIBASE]]/[[MULTICODEC]] representation for the public key, <code>ed25519-pub</code>,
and the representation for the private key, <code>ed25519-priv</code>, are shown below.
```

File: index.html
Comment (TallTed) [inline index.html]:
```suggestion
Signing begins with a credential without an attached proof, which is converted 
to canonical form, and then hashed, as shown in the following three examples.
```

File: index.html
Comment (TallTed) [inline index.html]:
```suggestion
and obtain its hash, as shown in the next three examples.
```

File: index.html
Comment (TallTed) [inline index.html]:
```suggestion
Finally, we combine the two hashes, use the private key with the combined hash to
```

File: index.html
Comment (TallTed) [inline index.html]:
```suggestion
      <p>Assemble the signed credential with the following two steps:</p>
      <ol>
        <li>
Add the <code>proofValue</code> field with the previously computed base58-btc 
value to the proof options document.
        </li>
        <li>
Set the <code>proof</code> field of the credential to the augmented proof 
option document.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Small stuff.

File: index.html
Comment (dlongley) [inline index.html:823]: We should probably have `@context` here and `"type": "Multikey"`.

**Review by dlongley — APPROVED** [review]
LGTM as a clear improvement over what was previously there. I haven't checked the test vector against an implementation myself.

File: index.html
Comment (msporny) [inline index.html:940]: Not for this PR, but a future PR should update this to use the v2 context (the example will look much cleaner, IMHO).

Comment (Wind4Greg) [reply index.html:940]: Will do. I'm currently working on test vectors for the "DataIntegrityProof" representation so will make sure to use the latest context!

File: index.html
Comment (msporny) [inline index.html:957]: Also for a future PR, but we should switch to the use of `cryptosuite` and `eddsa-2022`.

Comment (Wind4Greg) [reply index.html:957]: Hmm, the type "Ed25519Signature2020" doesn't have a `cryptosuite` parameter. The type "DataIntegrityProof" does. I'm working on the type "DataIntegrityProof" still.

Comment (Wind4Greg) [reply index.html:957]: Hmm, the type "Ed25519Signature2020" doesn't have a `cryptosuite` parameter. The type "DataIntegrityProof" does. I'm working on the type "DataIntegrityProof" still.

Comment (msporny) [reply index.html:957]: Yep, sorry, I meant... switch to `type` of `DataIntegrityProof` with `cryptosuite` set to `eddsa-2022`.

File: index.html
Comment (OR13) [inline index.html:863]: this escaping makes my eyes bleed, prefer a `data-include`

File: index.html
Comment (OR13) [inline index.html:939]: I suggest your test vectors NOT require loading 3 other contexts....

File: index.html
Comment (Wind4Greg) [reply index.html:863]: Thanks! I'm new to ReSpec. `data-include` should make bringing in the test vectors much easier.

File: index.html
Comment (Wind4Greg) [reply index.html:939]: Agree! I posted an issue at https://github.com/w3c/vc-di-eddsa/issues/27. Looking for feedback on the example unsigned document (we've got three different test vector cases to produce). I was suggesting the following which only depends on one context. Also I posted the code that I'll use to generate the test vectors at https://github.com/Wind4Greg/EdDSA-Test-Vectors.

```json
{
    "@context": [
        "https://www.w3.org/ns/credentials/v2",
        {"UniversityDegreeCredential": "https://example.org/examples#UniversityDegreeCredential"}
    ],
    "id": "http://example.edu/credentials/3732",
    "type": [
        "VerifiableCredential",
        "UniversityDegreeCredential"
    ],
    "issuer": "https://example.edu/issuers/565049",
    "issuanceDate": "2010-01-01T00:00:00Z",
    "credentialSubject": {
        "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
        "degree": {
            "type": "BachelorDegree",
            "name": "Bachelor of Science and Arts"
        }
    }
}
```

File: index.html
Comment (msporny) [reply index.html:863]: I think `data-include` breaks PR-preview, unless they recently fixed that... not saying we shouldn't use it, but something to consider.

File: index.html
Comment (msporny) [reply index.html:823]: We should probably just break these values out and not have them in a JSON data structure that we don't want people producing in the real world. I'll do that after the merge. We haven't specified how / where to define `privateKeyMultibase` and we don't want that perma-thread holding up this PR.

File: index.html
Comment (msporny) [reply index.html:939]: The VC above is problematic... `UniversityDegreeCredential` is defined, but the following items are going to be converted to issuer-defined properties, which might confuse some people: issuanceDate, degree, BachelorDegree. Let's keep the current examples and then simplify later. We should probably just shift to a simple hello world example using only the `credentials/v2` context... which does define properties like "name" and "description".

**Comment by msporny** [comment]
Editorial (test vectors are editorial), multiple reviews, changes requested (some made now, some will be made after the merge), no objections, merging.

File: index.html
Comment (Wind4Greg) [reply index.html:939]: Sounds good.

**Pull request merged**
